CMakeLists.txt

```diff
@@ -455,7 +455,9 @@ else()
     endif()
     unset(compiler_supports_ftrapv)
 
-    string(PREPEND CMAKE_CXX_FLAGS_DEBUG "-O0 ")
+    if(NOT CMAKE_CXX_FLAGS_DEBUG MATCHES "(^| )-O[^ ]*")
```

Comment:
Why is this needed? (also why is it only needed now). Concecp NACK generally on having to use regexes like this. Seems to me this is working around some other problem?

Reply:
Going to remove that change.

Reply:
Thanks! [Deleted](https://github.com/hebasto/bitcoin/pull/93#issuecomment-2130035240).